frontend/src/features/manageVehicles/hooks/endpoints/endpoints.tsx

```diff
@@ -19,10 +19,10 @@ export const MV_BACKEND_URL =
   ENV === "localhost"
     ? "http://localhost:5000"
     : ENV === ("test" || "uat" || "prod")
-    ? `https://onroutebc-${ENV}-backend.apps.silver.devops.gov.bc.ca`
+    ? `https://onroutebc-${ENV}-backend-vehicles.apps.silver.devops.gov.bc.ca`
```

Comment:
I believe, adding more subdomains means, we'll have to acquire security certificates for each such subdomain. We might want to (eventually) do away with this and try to push all the apis under one subdomain. We had this issue with formsflow elsewhere.

Reply:
Made a note for future consideration